inc/real_guids/namespace.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Altis\CMS\Real_GUIDs;
+
+function bootstrap() {
```

Comment:
Missing docblock here:
```suggestion
/**
 * Add filters for setting real GUIDs.
 */
function bootstrap() {
```